Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/LiveDataExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package de.rki.coronawarnapp.util.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+
+fun <T> LiveData<T>.observe2(fragment: Fragment, callback: (T) -> Unit) {
```

Comment:
function names with numbers are not great to read imo and we should try to make the naming more explicit

Reply:
This is not necessary to be used, but codesugar that makes code look much nicer. I usually use this naming to denote, that a function is functionally the same, but an upgrade. Autocompletion would show this variant to someone trying to use `.observe`.